assets/cubyz/biomes/trans/beach.zig.zon

```diff
@@ -0,0 +1,33 @@
+.{
+	.properties = .{
+		.ocean,
+		.wet,
+	},
+	.radius = 128,
```

Comment:
Transition biomes don't need a radius, only sub and normal biomes do, let's not add extra noise with these unnecessary parameters.